2022-01-20 21:32:49,626 INFO    Thread-57 :224864 [wandb_setup.py:_flush():71] setting env: {'entity': 'amazonproject', 'project': 'Amazon-Reviews-v2', 'root_dir': '/zhome/4a/f/117237/Documents/MLOps_Amazon_review_project', 'run_id': 'moy88zb1', 'sweep_param_path': '/zhome/4a/f/117237/Documents/MLOps_Amazon_review_project/wandb/sweep-igrmn63w/config-moy88zb1.yaml', 'sweep_id': 'igrmn63w'}
2022-01-20 21:32:49,626 INFO    Thread-57 :224864 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 21:32:49,627 INFO    Thread-57 :224864 [wandb_init.py:_log_setup():371] Logging user logs to /zhome/4a/f/117237/Documents/MLOps_Amazon_review_project/wandb/run-20220120_213249-moy88zb1/logs/debug.log
2022-01-20 21:32:49,627 INFO    Thread-57 :224864 [wandb_init.py:_log_setup():372] Logging internal logs to /zhome/4a/f/117237/Documents/MLOps_Amazon_review_project/wandb/run-20220120_213249-moy88zb1/logs/debug-internal.log
2022-01-20 21:32:49,627 INFO    Thread-57 :224864 [wandb_init.py:init():404] calling init triggers
2022-01-20 21:32:49,627 INFO    Thread-57 :224864 [wandb_init.py:init():411] wandb.init called with sweep_config: {'batch_size': 100, 'drop_out': 0.35, 'lr': 0.001, 'optimizer': 'sgd'}
config: {}
2022-01-20 21:32:49,628 INFO    Thread-57 :224864 [wandb_init.py:init():460] starting backend
2022-01-20 21:32:49,628 INFO    Thread-57 :224864 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-20 21:32:49,636 INFO    Thread-57 :224864 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 21:32:49,654 INFO    Thread-57 :224864 [backend.py:ensure_launched():222] started backend process with pid: 234243
2022-01-20 21:32:49,657 INFO    Thread-57 :224864 [wandb_init.py:init():469] backend started and connected
2022-01-20 21:32:49,661 INFO    Thread-57 :224864 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 100, 'drop_out': 0.35, 'lr': 0.001, 'optimizer': 'sgd'}
2022-01-20 21:32:49,664 INFO    Thread-57 :224864 [wandb_init.py:init():533] updated telemetry
2022-01-20 21:32:49,734 INFO    Thread-57 :224864 [wandb_init.py:init():563] communicating current version
2022-01-20 21:32:50,926 INFO    Thread-57 :224864 [wandb_init.py:init():568] got version response 
2022-01-20 21:32:50,927 INFO    Thread-57 :224864 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 21:32:51,167 INFO    Thread-57 :224864 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 21:32:51,240 INFO    Thread-57 :224864 [wandb_run.py:_console_start():1810] atexit reg
2022-01-20 21:32:51,246 INFO    Thread-57 :224864 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-20 21:32:51,246 INFO    Thread-57 :224864 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-20 21:32:51,246 ERROR   Thread-57 :224864 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/zhome/4a/f/117237/.local/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/zhome/4a/f/117237/.local/lib/python3.6/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/zhome/4a/f/117237/.local/lib/python3.6/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/zhome/4a/f/117237/.local/lib/python3.6/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib64/python3.6/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-20 21:32:51,246 INFO    Thread-57 :224864 [wandb_init.py:init():633] run started, returning control to user process
2022-01-20 21:47:12,091 INFO    Thread-57 :224864 [wandb_run.py:finish():1457] finishing run amazonproject/Amazon-Reviews-v2/moy88zb1
2022-01-20 21:47:12,091 INFO    Thread-57 :224864 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-20 21:47:12,094 INFO    Thread-57 :224864 [wandb_run.py:_restore():1752] restore
2022-01-20 21:47:12,103 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 809
  total_bytes: 809
}

2022-01-20 21:47:12,382 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 809
  total_bytes: 809
}

2022-01-20 21:47:12,707 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 809
  total_bytes: 809
}

2022-01-20 21:47:12,809 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 809
  total_bytes: 2622
}

2022-01-20 21:47:12,911 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 809
  total_bytes: 2622
}

2022-01-20 21:47:13,013 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2622
  total_bytes: 2622
}

2022-01-20 21:47:13,114 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2622
  total_bytes: 2622
}

2022-01-20 21:47:13,217 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2622
  total_bytes: 2622
}

2022-01-20 21:47:13,319 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2622
  total_bytes: 2622
}

2022-01-20 21:47:13,421 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2622
  total_bytes: 2622
}

2022-01-20 21:47:13,522 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2622
  total_bytes: 2622
}

2022-01-20 21:47:14,034 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2622
  total_bytes: 2622
}

2022-01-20 21:47:14,499 INFO    Thread-57 :224864 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 2622
  total_bytes: 2622
}
local_info {
}

2022-01-20 21:47:15,605 INFO    Thread-57 :224864 [wandb_run.py:_append_history():2130] rendering history
2022-01-20 21:47:15,605 INFO    Thread-57 :224864 [wandb_run.py:_append_summary():2085] rendering summary
2022-01-20 21:47:15,605 INFO    Thread-57 :224864 [wandb_run.py:_append_files():2180] logging synced files
